### PR TITLE
ns-web-vm removal of tagging & upgrade to 5.3.0

### PR DESCRIPTION
**Pull request by kennethgillen**
Tested on infra-testpr (connecting to eel), with success.

```
PLAY RECAP *********************************************************************
infra-testpr.openmicroscopy.org : ok=65   changed=11   unreachable=0    failed=0
```

[infra-testpr_ns-web_to_530.txt](https://github.com/openmicroscopy/infrastructure/files/919519/infra-testpr_ns-web_to_530.txt)

REQUIRES: https://github.com/openmicroscopy/management_tools/pull/373

### CODE REVIEW

File: ansible/server-state-playbooks/nightshade-web/playbook.yml [hunk 1]
```diff
@@ -149,18 +151,6 @@
...
-      become: yes
-      unarchive:
-        src: "http://downloads.openmicroscopy.org/webtagging/{{ omero_web_extensions_tagging_ver }}/webtagging-{{ omero_web_extensions_tagging_ver }}.zip"
-        owner: "root" 
-        group: "root" 
-        mode: "u=rwx,go=rx" 
-        dest: "{{ omero_web_extensionsdir }}"
-        remote_src: True
-        creates: "{{ omero_web_extensionsdir }}/webtagging-{{omero_web_extensions_tagging_ver}}" 
-
```
Comment (manics) [inline ansible/server-state-playbooks/nightshade-web/playbook.yml]: I guess you're aware this won't delete existing files if run on a VM that was already deployed with the old playbook.

Comment (kennethgillen) [inline ansible/server-state-playbooks/nightshade-web/playbook.yml]: Yep, thanks. It does remove the relevant lines from the OMERO.web config though, and restarts web/re-creates the config, which makes it take effect.

**Comment by kennethgillen** [comment]
Run against ns-web: 
[ns-web-530_upgrade-provision_20170418-163410Z.txt](https://github.com/openmicroscopy/infrastructure/files/930492/ns-web-530_upgrade-provision_20170418-163410Z.txt)

Matching private config at https://github.com/openmicroscopy/management_tools/pull/373

**Comment by joshmoore** [comment]
Final changes will be pushed here before merging.

**Comment by kennethgillen** [comment]
After performance PR added above, (which made it to master while this branch was open) re-run at 18:35UTC

Check-mode: 
[ns-web-530_upgrade-provision_20170418-183327Z.txt](https://github.com/openmicroscopy/infrastructure/files/933400/ns-web-530_upgrade-provision_20170418-183327Z.txt)

Run mode: 
[ns-web-530_upgrade-provision_20170418-183538Z.txt](https://github.com/openmicroscopy/infrastructure/files/933403/ns-web-530_upgrade-provision_20170418-183538Z.txt)

Then manual modification made to VM (cores 4->2), re-run the Ansible because it calculates the NGINX workers based on total core count:

Check-mode: 
[ns-web-530_upgrade-provision_20170418-230630Z.txt](https://github.com/openmicroscopy/infrastructure/files/933407/ns-web-530_upgrade-provision_20170418-230630Z.txt)

Run mode: 
[ns-web-530_upgrade-provision_20170418-230639Z.txt](https://github.com/openmicroscopy/infrastructure/files/933409/ns-web-530_upgrade-provision_20170418-230639Z.txt)

**Comment by joshmoore** [comment]
After updates, merging as the current state of `ns-web`. Thanks, @kennethgillen

**Pull request merged**